src/psyclone/tests/psyir/nodes/assignment_test.py

```diff
@@ -129,3 +129,32 @@ def test_assignment_children_validation():
         assignment.children.append(Literal("0.0", REAL_SINGLE_TYPE))
     assert ("Item 'Literal' can't be child 2 of 'Assignment'. The valid format"
             " is: 'DataNode, DataNode'.") in str(excinfo.value)
+
+
+def test_is_array_range():
+    '''test that the is_array_range method behaves as expected, returning
+    true if the LHS of the assignment is an array range access and
+    false otherwise.
+
+    '''
+    one = Literal("1.0", REAL_TYPE)
+    int_one = Literal("1", INTEGER_TYPE)
+    var = DataSymbol("x", REAL_TYPE)
+    reference = Reference(var)
+
+    # lhs is not an array
+    assignment = Assignment.create(reference, one)
+    assert assignment.is_array_range == False
+
+    # lhs is an array reference but has no range
+    array_type = ArrayType(REAL_TYPE, [10,10])
```

Comment:
pylint complains about missing whitespace here and on 152.

Reply:
Fixed.